server/data/users.js

```diff
@@ -67,75 +67,9 @@ const getUserByEmail = async (email) => {
   return user;
 };
 
```

Comment:
Why this is removed if this from merge conflicts make sure all changes from others are included